Moderation-Policy.md

```diff
@@ -296,19 +296,22 @@ remove resigning team member from respective permissions and private access.
 ### Current Members of Moderation Team
 * [aduh95](https://github.com/aduh95) -
 **Antoine du Hamel** <<duhamelantoine1995@gmail.com>> (he/him)
+* [atlowChemi](https://github.com/atlowChemi) -
+**Chemi Atlow** <<chemi@atlow.co.il>> (he/him)
 * [benjamingr](https://github.com/benjamingr) -
 **Benjamin Gruenbaum** &lt;benjamingr@gmail.com&gt;
-* [JohnTitor](https://github.com/JohnTitor) -
-**Yuki Okushi** &lt;huyuumi.dev@gmail.com&gt;
+* [bmuenzenmeyer](https://github.com/bmuenzenmeyer) -
+**Brian Muenzenmeyer** <<brian.muenzenmeyer@gmail.com>> (he/him)
 * [othiym23](https://github.com/othiym23) -
 **Forrest L Norvell** &lt;othiym23@gmail.com&gt;
 * [ovflowd](https://github.com/ovflowd) -
 **Claudio Wunder** &lt;cwunder@gnome.org&gt; (he/they)
-* [ryanmurakami](https://github.com/ryanmurakami) -
-**Ryan Lewis** &lt;ryanharrisonlewis@gmail.com&gt;
+* [JohnTitor](https://github.com/JohnTitor) -
+**Yuki Okushi** &lt;huyuumi.dev@gmail.com&gt;
```

Comment:
Can you move that back up so the list stays in case-insensitive alphabetical-order please?

Reply:
Isn't it ASCII order?

Reply:
IMO, for GitHub usernames, case-insensitive order makes more sense from the perspective of the human user. If I'm scanning a long list to see if "Trott" is in there, I probably don't care whether it's "Trott" or "trott" and I might or might not think to check for both. So having both those terms end up in the same place is good usability.

Case-sensitive would make sense if "Trott" and "trott" could conceivably be different people, but they can't. GitHub names are case insensitive, so our list should be too.

Case-sensitive, arguably, also optimizes for tooling.

Those last two items are why we order bottom references in documentation as case-sensitive ordering. Capitalization does matter there (I think) and making things slightly easier for tooling is good too. (If you have "example.com" and "Example.com", which one comes first? If we're doing case-sensitive ordering, there's no decision to make there. So that's good.)

It's not great that we have a lot of things that are case-insensitive ordered and one (or maybe more than one) thing ordered case-sensitive, but everything is a tradeoff.

Reply:
Case-insensitive is how we order the similar lists in nodejs/node's `README.md`. I agree with what Trott said, having case-insensitive makes it easier for humans to browse the list looking for someone. We could reuse the linter we use on the other repo here so that discussion doesn't hold future changes

Reply:
I opened https://github.com/nodejs/admin/pull/892 to add the linter.